blog/docs/releases/announcing-knative-v1-4-release.md

```diff
@@ -0,0 +1,209 @@
+- Reduced duplication of tracing.yaml config map which existed in multiple locations by symlinking "config/channels/in-memory-channel/configmaps/tracing.yaml" to "config/core/configmaps/tracing.yaml". (#6295, @xtreme-sameer-vohra)
+- The `sendevent` tool is not part of Eventing core anymore, an alternative is available as `kn` plugin https://github.com/knative-sandbox/kn-plugin-event (#6284, @pierDipi)
+
+### 🐞 Bug Fixes
+
+ - Set dead letter sink URI in the Channel status (#6256, @pierDipi)
+ - MTChannelBroker can now co-exist with other broker implementations. (#6265, @n3wscott)
+ - The service created for the inmemorychannel's dispatcher will have ports defined, similar to the rest of the services. (#6283, @jhill072)
+
+## Eventing Extensions
+
+### Apache Kafka Broker v1.4
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-kafka-broker/releases/tag/knative-v1.4.0 -->
+
+#### 💫 New Features & Changes
+
+ - Handle host headers in dataplane for identification of channel instances (#1990, @aliok)
+ - There is now auto-migration from old consolidated KafkaChannel to the new KafkaChannel. Low level configuration options such as Sarama settings are not migrated to the new channel. However, channel url and auth settings are migrated. (#2004, @aliok)
+ - Discard Consumer records that are not CloudEvents (#2066, @aavarghese)
+ - Dynamically set max.poll.interval.ms based on the delivery spec (#2058, @pierDipi)
+ - The default request timeout is 10 minutes now, it was previously set to 10 seconds.
+ - It can be overridden using spec.delivery.timeout on Broker, Trigger, KafkaChannel and Subscription. (#2057, @pierDipi)
+ - Add new new-trigger-filters experimental feature. When enabled, Triggers support a new filters field that conforms to the filters API field defined in the CloudEvents Subscriptions API. It allows you to specify a set of powerful filter expressions, where each expression evaluates to either true or false for each event. (#1922, @devguyio)
+
+
+### RabbitMQ Broker and Source v1.4
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.4.0 -->
+
+
+#### 💫 New Features & Changes
+
+- Now performance graphs for the RabbitMQ Broker are going to be generated for every mayor version release inside our performance test directories (#668, @gabo1208)
+- Now the RabbitMQ Source, when failing to send an event, does not get stuck in a cycle requeueing messages as they come. It follows a retries based backoff strategy (and does not reqeue them if the prefetchCount is 1) (#614, @gabo1208)
+- Now the RabbitMQ's Broker Ingress pod can recover from a closed Channel or Connection (#648, @gabo1208)
+- 🧹 Changed prefetchCount env variable name to Parallelism in both Trigger and Source, to better reflect its functionality (#676, @gabo1208)
+- Remove support for managed RabbitMQ brokers (#708, @gab-satchi)
```

Comment:
Can this be under a `🚨 Breaking or Notable Changes` section as its removing existing functionality?
```suggestion


#### 🚨 Breaking or Notable Changes

- Remove support for managed RabbitMQ brokers (#708, @gab-satchi)
```

Reply:
done